src/main/cpp/kernels/SystemML.cu

```diff
@@ -20,7 +20,7 @@
 /**********************************
 When updating a kernel or adding a new one,
 please compile the ptx file and commit it:
-nvcc -ptx -arch=sm_30 SystemML.cu
+nvcc -ptx -arch=sm_30 --std c++11 SystemML.cu
```

Comment:
There is makefile in this directory. Please update that with the C++11 flag too.

Reply:
Done.